app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx

```diff
@@ -1,24 +1,22 @@
 import React, { useState } from "react";
 
-import { Alert } from "neetoui";
-
-import notesApi from "apis/notes";
+import { Alert, Toastr } from "neetoui";
 
 const DeleteAlert = ({
-  refetch,
   onClose,
-  selectedNoteIds,
-  setSelectedNoteIds,
+  selectedNoteId,
+  setNotes,
+  setSelectedNoteId,
 }) => {
   const [deleting, setDeleting] = useState(false);
 
   const handleDelete = async () => {
     try {
       setDeleting(true);
-      await notesApi.destroy({ ids: selectedNoteIds });
+      setNotes(oldNotes => oldNotes.filter(note => note.id !== selectedNoteId));
       onClose();
-      setSelectedNoteIds([]);
-      refetch();
+      setSelectedNoteId();
```

Comment:
We should never invoke a setter without any values. It's an antipattern. Either this setter is not required or you forgot to send the value in the setter.